tests/tamr_client/datasets/test_record.py

```diff
@@ -154,6 +155,20 @@ def test_delete_infer_primary_key():
     assert snoop["payload"] == utils.stringify(deletes)
 
 
+@responses.activate
+def test_stream():
+    s = utils.session()
+    dataset = utils.dataset()
+
+    url = tc.URL(path="datasets/1/records")
+    responses.add(
+        responses.GET, str(url), body="\n".join([json.dumps(x) for x in _records_json])
+    )
+
+    response = tc.record.stream(s, dataset)
```

Comment:
nit: `response` connotes that this object is an HTTP response. You could just use `records` instead:

```suggestion
    records = tc.record.stream(s, dataset)
```